.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # Specify the Ruby version you desire here
+      - image: circleci/python:3.6.2
```

Comment:
this is like a python 2-to-3 upgrade? I would be *very* wary of doing this just to get CI working -- have this been tested locally with the SP? Do we have this deployed in production anywhere that we need to worry about upgrading?

Reply:
I haven't been able to successfully get `pip install -r requirements.txt` to work locally with Python 2.7, so the upgrade to Python 3 was not just for Circle CI. I don't think we have this deployed in production. 

The cert and key were also invalid. I updated them just now [here](https://github.com/18F/identity-sp-python/pull/2/commits/07c289fd279a1df34f6881b319eec4cfce2fd0a5) and tested locally with Python 3 and I was able to sign in successfully. Logout doesn't work, but I don't think it worked previously either.  We can fix logout later.

Reply:
ok thanks!